File: pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

//American coins: 25c, 10c, 5c, 1c

int main(void)
{
// Declaring the variables
float cash;
int variable = 0; //int variable is the number of times there is a change

// Prompt the user's input for the money he/she wants to change until it is a positive number
do
    {cash = get_float("Gimme your money!\n");
    printf("Change owned: %0.2f\n", cash);
    }
while (cash <= 0);

// Eliminates the decimal number from the float cash as Integer c
int c = round(cash * 100);

// Grand loop starts
while (c - 25 >= 0) //minus all 25cents possible
   {c = c - 25;
    variable++;
   }

{while (c - 10 >= 0) //minus all 10cents possible
   {c = c - 10;
    variable++;
   }

while (c - 5 >= 0) //minus all 5cents possible
   {c = c - 5;
    variable++;
   }

while (c - 1 >= 0) //minus all 1cents possible
   {c = c - 1;
    variable++;
   }

printf("%i\n", variable);
}
}

File: pset6/cash/cash.py
from cs50 import get_float, get_int, get_string

# Declaring the variables
# variable is the number of times there is a change
cash = 0.00
variable = 0

# Prompt the user's input for the money he/she wants to change until it is a positive number
while cash <= 0:
    cash = get_float("Gimme your money!\n")
    print("Change owed: " + str(cash))

# Eliminates the decimal number from the float cash as Integer c
c = round(cash * 100)

#minus all 25 cents possible
while c - 25 >= 0:
    c -= 25
    variable += 1

#minus all 10 cents possible
while c - 10 >= 0:
    c -= 10
    variable += 1

#minus all 5 cents possible
while c - 5 >= 0:
    c -= 5
    variable += 1

#minus all 1 cents possible
while c - 1 >= 0:
    c -= 1
    variable += 1

print(variable)
